src/test/resources/ibanity.properties

```diff
@@ -37,3 +37,11 @@ ibanity.tpp.account-information-access-result.redirect-url=https://fake-tpp.com/
 # URI where Ibanity must use to generate a redirect response when the end-user finalises
 #   the payments on the Financial Institutions side
 ibanity.tpp.payment-initiation-result.redirect-url=https://fake-tpp.ibanity.com/payment_redirection
+
+# URI where Ibanity must use to generate a redirect response when the end-user finalises
+#   the oauth2 flow in ponto-connect
+ibanity.ponto-connect.oauth2.redirect-url=https://localhost:8080/oauth2/callback
+ibanity.ponto-connect.oauth2.client_id=<client_id>
+ibanity.ponto-connect.oauth2.client_secret=<client_secret>
+ibanity.ponto-connect.oauth2.code_verifier=<code_verifier>
```

Comment:
Why is code_verifier in the config?

Reply:
ibanity.properties is not the config but a property file used by the sample clients.
code_verifier is externalized as it's an input value to start performing sample calls.